frontend/src/contexts/HostContext/hostReducerMiddleware.spec.js

```diff
@@ -98,6 +117,27 @@ describe('hostReducerMiddleware', () => {
     });
   });
 
+  describe('PLAYER_DISCONNECTED', () => {
+    it("calls socketInstane's sendMessage with a remove disconnected players card message", () => {
```

Comment:
I won’t block this but there is a small typo here

Reply:
Thanks for the heads up!